tests/test_browser_model.py

```diff
@@ -11,17 +10,22 @@ class TestBrowserModel(object):
     def test_new_private_window(self, selenium):
         """Tests opening a new private browsing window via menu"""
         foxpuppet = FoxPuppet(selenium)
+        wm = foxpuppet.window_manager
+        print(wm.windows)
+        assert len(wm.windows) == 1
         foxpuppet.browser.open_window(private=True)
-        assert len(foxpuppet.windows.all) == 2
-        assert foxpuppet.browser.is_private is False
-        foxpuppet.windows.focus(foxpuppet.windows.all[1])
-        assert foxpuppet.browser.is_private is True
+        assert len(wm.windows) == 2
+        wm.windows[1].focus()
+        assert foxpuppet.browser.is_private
```

Comment:
I feel that `foxpuppet.browser` should be the initial browser window, and shouldn't follow whatever is the currently focused window. We should therefore have an assert like `assert new_window is not foxpuppet.browser` and the private check should be `assert new_window.is_private`.